core/tabs/system-setup/fedora/rpm-fusion-setup.sh

```diff
@@ -13,7 +13,10 @@ installRPMFusion() {
                 "$ESCALATION_TOOL" "$PACKAGER" config-manager --enable fedora-cisco-openh264
                 "$ESCALATION_TOOL" "$PACKAGER" config-manager --set-enabled rpmfusion-nonfree-updates
                 "$ESCALATION_TOOL" "$PACKAGER" config-manager --set-enabled rpmfusion-free-updates
-                printf "%b\n" "${GREEN}RPM Fusion installed and enabled${RC}"
+                "$ESCALATION_TOOL" "$PACKAGER" install rpmfusion-free-release-tainted rpmfusion-nonfree-release-tainted -y
+                "$ESCALATION_TOOL" "$PACKAGER" config-manager --set-enabled rpmfusion-free-release-tainted 
```

Comment:
```suggestion
                "$ESCALATION_TOOL" "$PACKAGER" config-manager --set-enabled rpmfusion-free-release-tainted
```

CI failing because of trailing space